Assets/Sqripts/Player.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Player : MonoBehaviour
+{
+    [SerializeField] private Rigidbody2D _rigidbody;
+    [SerializeField] private float _speed;
+    [SerializeField] private float _jumpforce;
+    [SerializeField] private CheckGround _isGround;
+    [SerializeField] private Coin _coin;
+    [SerializeField] private GameObject _textCoin;
+    void Start()
+    {
+        _rigidbody = GetComponent<Rigidbody2D>();
+        _isGround = GetComponent<CheckGround>();
+        _coin = _textCoin.GetComponent<Coin>();
+    }
+
+    void Update()
+    {
+        float Horizontal = Input.GetAxisRaw("Horizontal");
+        _rigidbody.velocity=new Vector2(Horizontal* _speed, _rigidbody.velocity.y);
+
+        if (Input.GetAxisRaw("Vertical")==1)
+        {
+            if (_isGround._CheckGround())
+            {
+                _rigidbody.AddForce(Vector2.up * _jumpforce, ForceMode2D.Impulse);
+            }
+        }
+    }
+
+    private void OnTriggerEnter2D(Collider2D collision)
+    {
+        if (collision.CompareTag("Coin"))
```

Comment:
https://trello.com/c/LCsg3Gnz/2-%D1%80%D0%B0%D0%B1%D0%BE%D1%82%D0%B0-%D1%81-%D1%82%D0%B5%D0%B3%D0%B0%D0%BC%D0%B8